enzy_htp/_interface/pymol_interface.py

```diff
@@ -279,12 +279,22 @@ def export_pdb(self, pymol_session: pymol2.PyMOL, pymol_obj_name: str, if_retain
             pymol_session.cmd.set("retain_order")
         pymol_session.cmd.save(pymol_outfile_path, pymol_obj_name)
 
+        # fix the atom naming issue.
+        temp_stru = PDBParser().get_structure(pymol_outfile_path,allow_multichain_in_atom=True)
+        temp_stru.fix_pymol_naming()
```

Comment:
Naming should not be fixed here since the naming error is not caused by the general file saving behavior of PyMol but by those pymol functions that generates new structures. (e.g.: point mutation)
But this design is not easy since the point_mutate function do not have access to the naming change (or at least we need to write new interface for it). I can suggest to new design here:
1. move it to export_enzy_htp_stru() and turning it off by default (manually turn it on when calling from the mutation api)
2. change the fix_pymol_naming by changing the atom names using pymol's `alter` and call it in point_mutate